harbor_registry - jfrog - update failed

**Describe the bug**
when trying to update a harbor_registry of type jfrog it fails
because of CredentialType : https://github.com/goharbor/terraform-provider-harbor/blob/main/client/registry.go#L31
which is set to "jfrog-artifactory" instead of "basic"

in the core logs : 
```
2023-08-22T14:42:14Z [ERROR] [/lib/http/error.go:56]: {"errors":[{"code":"UNKNOWN","message":"unknown: ERROR: value too long for type character varying(16) (SQLSTATE 22001)"}]}
```

we don't need to set CredentialType.

--- a/client/registry.go
+++ b/client/registry.go
@@ -24,16 +24,14 @@ func GetRegistryBody(d *schema.ResourceData) models.RegistryBody {
 }
 
 func GetRegistryUpdateBody(d *schema.ResourceData) models.RegistryUpdateBody {
-	regType, _ := GetRegistryType(d.Get("provider_name").(string))
 
 	body := models.RegistryUpdateBody{
-		AccessKey:      d.Get("access_id").(string),
-		CredentialType: regType,
-		AccessSecret:   d.Get("access_secret").(string),
-		Description:    d.Get("description").(string),
-		Insecure:       d.Get("insecure").(bool),
-		Name:           d.Get("name").(string),
-		URL:            d.Get("endpoint_url").(string),
+		AccessKey:    d.Get("access_id").(string),
+		AccessSecret: d.Get("access_secret").(string),
+		Description:  d.Get("description").(string),
+		Insecure:     d.Get("insecure").(bool),
+		Name:         d.Get("name").(string),
+		URL:          d.Get("endpoint_url").(string),
 	}
 
 	return body

